processor/elasticinframetricsprocessor/testdata/config.yaml

```diff
@@ -1,8 +1,12 @@
-default:
+elasticinframetrics:
+elasticinframetrics/2:
```

Comment:
I would change it to /all (and not /2 ). WDYT?

Reply:
done https://github.com/elastic/opentelemetry-collector-components/pull/14/commits/cf215e8e1fe25354459ee6e668f97a1e9f170a94